src/coreclr/src/vm/eventpipe.cpp

```diff
@@ -100,9 +101,18 @@ void EventPipe::Initialize()
         if (tracingInitialized)
             s_state = EventPipeState::Initialized;
     }
-#ifdef FEATURE_EVENTPIPE_STARTUP
     EnableViaEnvironmentVariables();
-#endif // FEATURE_EVENTPIPE_STARTUP
+}
+
+// Finish setting up the rest of EventPipe.
+void EventPipe::FinishSetup()
```

Comment:
nit: Can we call this `FinishInitialize` so it matches `EventPipe::Initialize` and the other functions in `main`?